### DISCUSSION TITLE
hurlfmt check always returns non-zero exit code

### DISCUSSION THREAD

**Discussion by azzamsa** [Orange-OpenSource/hurl#2917] [Q&A] [answered]
Hi, 👋🏽 

I am using `hurlfmt 4.3.0` and when running ` hurlfmt tests/api-collection/meta/health.hurl --check` it always gives me non-zero exit status. However  `hurlfmt tests/api-collection/meta/health.hurl --in-place` doesn't change or complain about anything.

``````hurl
POST {{base_url}}
Content-Type: application/json

```graphql
{
  health {
    status
  }
}
```

HTTP 200
[Asserts]
status == 200
jsonpath "$.data.health.status" == "running"
``````

```bash
tin on  master [!] is 📦 v1.2.0 via 🦀 v1.78.0
🐡 ❯ hurlfmt tests/api-collection/meta/health.hurl --check

tin on  master [!] is 📦 v1.2.0 via 🦀 v1.78.0
🐡 ❯ echo $status
1
````

Am I missing something?

Thanks! 🍜

**Comment by jcamiel** [comment, marked as answer]
Hi @azzamsa thanks for this discussion. I think this is kind of a bug: with `--check`, we return a status code of 1 whether the format is good or not. We should fix it so I'm converting this discussion to an issue.
